python/sensitive_detectors.py

```diff
@@ -29,6 +29,9 @@ def hcal() :
     def target() :
         return ScoringPlaneSD('target')
 
+    def trigscint() :
+        return ScoringPlaneSD('trigscint')
```

Comment:
Other TS collections are named something like `TrigScint*` or `TriggerPad*`. All the scoring plane SD does is capitalize the first letter and add the `ScoringPlaneHits` suffix to get the collection name, so I'm wondering if you want to rename the scoring plane SD here (and the scoring plane volumes in the GDML) so that the collections are named similar to other ones.

Reply:
yeah.. here i have adhered to the no-caps convention in the other systems. you can debate whether it's Ecal or ECal of course and i think sw settled on Ecal a long time ago. 

so your suggestion is to call it trigScint? i could do that. i'd need to update the gdml too i think. i will if we settle on that.

Reply:
Yes, that is my suggestion, but I won't be using the TS scoring plane hits, so if its not a noticeable issue, then I wouldn't worry about it.